src/main/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/types/function/ScalarFunction.java

```diff
@@ -51,6 +52,22 @@
     EXP(func(T(NUMBER)).to(T)),
     EXPM1(func(T(NUMBER)).to(T)),
     FLOOR(func(T(NUMBER)).to(T)),
+    IF(
+            func(BOOLEAN, T(STRING), T(STRING)).to(T),
```

Comment:
In the MySQL mannual, the IF require there paramaters. IF(expr1,expr2,expr3)
https://dev.mysql.com/doc/refman/8.0/en/control-flow-functions.html#function_if